lib/elixir/pages/Writing Documentation.md

```diff
@@ -74,17 +74,19 @@ Code comments are for developers reading the code. They are useful to mark impro
 
 In other words, documentation is required, code comments are optional.
 
-## Privacy
+## Hiding Internal Modules and Functions
 
-Elixir does not allow developers to document private functions. That's because private functions only exist inside the module that defines them and cannot be accessed externally.
+Libraries often have two levels of public interface: On one hand, they expose an API for external clients. But on the other hand, the modules and functions that conform the library often need to use other modules and functions which are not part of the external public interface.
+
+These are modules and functions that are technically public, but are really internal and thus should not have documentation for end users.
 
 Luckily, Elixir allows developers to hide modules and functions from the documentation. For example, one common practice for documenting internal behaviour is to set the `@moduledoc` attribute to false while documenting each function:
 
     defmodule MyApp.Hidden do
       @moduledoc false
 
       @doc """
-      This does something.
+      This documentation won't appear in the docs.
```

Comment:
Yup, it won't however, be listed. Maybe listing is the word we want to use here.